src/ts/uimanager.ts

```diff
@@ -396,30 +400,43 @@ export class UIManager {
     this.managerPlayerWrapper.clearEventHandlers();
   }
 }
+// For the lack of a better place, a wip helper to configure a UI language
+function configureUiLanguage(config: any): void {
+  if (config.locale !== undefined) {
```

Comment:
We usually use the `if (config.locale) { ...` pattern

Reply:
If no locale is set in the `UIConfig` then we should try to detect the language from the `Accept-Language` header.